### PR TITLE
sci-libs/libticonv: Version bump to 1.1.5, EAPI bump 5 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: sci-libs/libticonv/libticonv-1.1.5.ebuild [hunk 1]
```diff
@@ -21,18 +21,20 @@ DEPEND="${RDEPEND}
...
+src_prepare() {
+	default
+	eautoreconf
+}
+
 src_configure() {
-	econf \
-		$(use_enable static-libs static) \
-		$(use_enable iconv)
+	econf $(use_enable iconv)
```
Comment (jlec) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: Why did you drop the static-lib configure here? The USE is still in IUSE.

File: sci-libs/libticonv/libticonv-1.1.5.ebuild [hunk 2]
```diff
@@ -21,18 +21,20 @@ DEPEND="${RDEPEND}
...
 src_install() {
 	default
 	if use doc; then
-		dohtml docs/html/*
-		docinto charsets
-		dohtml docs/charsets/*
+		HTML_DOCS=( docs/html/. )
+		einstalldocs
+		dodoc -r docs/charsets
 	fi
```
Comment (SoapGentoo) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: shift the `if` block before `default`, then you can omit calling `einstalldocs`

File: sci-libs/libticonv/libticonv-1.1.5.ebuild [hunk 3]
```diff
@@ -21,18 +21,21 @@ DEPEND="${RDEPEND}
...
 src_install() {
-	default
 	if use doc; then
-		dohtml docs/html/*
-		docinto charsets
-		dohtml docs/charsets/*
+		HTML_DOCS=( docs/html/. )
+		dodoc -r docs/charsets
 	fi
-	use static-libs || rm -f "${D}"/usr/$(get_libdir)/${PN}.la
```
Comment (SoapGentoo) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: also check `.la` situation here please

File: sci-libs/libticonv/libticonv-1.1.5.ebuild [hunk 4]
```diff
@@ -21,18 +21,21 @@ DEPEND="${RDEPEND}
...
 }
 
 src_install() {
-	default
 	if use doc; then
-		dohtml docs/html/*
-		docinto charsets
-		dohtml docs/charsets/*
+		HTML_DOCS=( docs/html/. )
+		dodoc -r docs/charsets
```
Comment (SoapGentoo) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: could've also done `DOCS+=( docs/charsets )`, but your choice

Comment (g-braeunlich) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: Are you sure? I think if DOCS is unset einstalldocs searches for default files and I don't have to specify them manually. With dodoc I would just add additional docs.

Comment (SoapGentoo) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: absolutely, it's just that on line 22, you seem to be defining `DOCS`...

Comment (g-braeunlich) [inline sci-libs/libticonv/libticonv-1.1.5.ebuild]: Oh! You are right!

**Pull request closed without merging**